uservice_ccutter/server.py

```diff
@@ -32,7 +32,7 @@ def server(run_standalone=False):
     """
     # Add "/ccutter" for mapping behind api.lsst.codes
     app = APIFlask(name="uservice-ccutter",
-                   version="0.0.3",
+                   version="0.0.4",
```

Comment:
This version is already in setup.py. I don't know if it's worth just collapsing the version into a version file so it can be referenced everywhere. It's unfortunate that the `Dockerfile` also needs the version.